microsoft-edge/devtools-guide-chromium/whats-new/2023/11/devtools-119.md

```diff
@@ -0,0 +1,60 @@
+---
+title: What's new in DevTools (Microsoft Edge 119)
+description: "Updated text color for error pills under the Issues panel. Updated text color for tree items under the Application panel. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 11/02/2023
+---
+# What's New in DevTools (Microsoft Edge 119)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Updated text color for error pills under the Issues panel
+
+Before Microsoft Edge 119, users with low vision might face difficulties with the color contrast ratio for the error pills.<!-- todo: define -->  The contrast ratio between the foreground color and background color has been updated to meet Web Content Accessibility Guidelines (WCAG) 1.4.3 requirements:
+
+![Updated text color for error pills under the Issues panel](./devtools-119-images/dark-text-color-on-pills.png)
+
+<!-- 
+todo: why 1.4.3, it's at WCAG 3
+https://www.w3.org/WAI/search/?q=wcag+1.4.3
+-->
```

Comment:
@mikehoffms the error pill UI was in violation of [this](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html). our contrast ratio was less than 4.5:1 which made it difficult read the text on the error pills.

Reply:
reworded & linked